Cargo.toml

```diff
@@ -5,10 +5,10 @@ exclude = ["python"]
 resolver = "2"
 
 [workspace.dependencies]
-lance = { "version" = "=0.8.17", "features" = ["dynamodb"] }
-lance-index = { "version" = "=0.8.17" }
-lance-linalg = { "version" = "=0.8.17" }
-lance-testing = { "version" = "=0.8.17" }
+lance = { "version" = "=0.8.19", "features" = ["dynamodb"] }
+lance-index = { "version" = "=0.8.19" }
+lance-linalg = { "version" = "=0.8.19" }
+lance-testing = { "version" = "=0.8.19" }
```

Comment:
We should skip v0.8.19 and go straight to 0.8.20. 0.8.19 has a serious bug that corrupts tables.